store/reducers/groups.reducer.js

```diff
@@ -0,0 +1,155 @@
+import * as actions from '../actions/groups.actions';
+
+const initialState = {
+  type: null,
+  error: null,
+  groups: [],
+  group: null,
+  usersContacts: [],
+  comments: [],
+  comment: null,
+  geonames: [],
+  peopleGroups: [],
+  activities: [],
+};
+
+export default function groupsReducer(state = initialState, action) {
+  const newState = {
+    ...state,
+    type: action.type,
```

Comment:
Action type shouldn't be needed outside of the reducer. I think this may need some discussion of what is trying to be done here and what was the difficulty to be overcome.